root/static/scripts/relationship-editor/components/DialogPreview.js

```diff
@@ -219,7 +219,7 @@ const DialogPreview = (React.memo<PropsT>(({
       ) : newRelationship ? (
         relationshipPreview(newRelationship, 'add-relationship')
       ) : (
-        <p>
+        <p className="required-fields-note">
           {l('Please fill out all required fields.')}
         </p>
```

Comment:
I think this'd be enough:

```suggestion
        <p>
          {l('Please fill out all required (*) fields.')}
        </p>
```

Reply:
A doubt I have with that is that the \*s next to the fields are displayed with CSS (in order to position them more easily), so won't be announced by screen readers.  If we were to add `aria-required` on the inputs, which I can do but haven't done yet, then we would also want the \*s to remain visual-only so that we don't announce two different "required" indicators to the user.

Including the \* in this message where it will be announced might be confusing, then, because the user won't be able to find which fields have a \* next to them.

Reply:
Hmm, ok, I see that point, although I feel the screen reader user might be able to figure out that the aria required is the equivalent of that?

My worry here is that the asterisk *before* the error message is kinda confusing, but maybe it's just me.

Reply:
I was copying what https://tickets.metabrainz.org/browse/MBS-11847?focusedCommentId=66587&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-66587 used, which I see as being styled like a footnote (where the asterisk would normally come first).